src/modules/event/EventType.ts

```diff
@@ -0,0 +1,43 @@
+import { GraphQLBoolean, GraphQLObjectType, GraphQLString } from "graphql";
+
+import { globalIdField } from "graphql-relay";
+
+import { connectionDefinitions } from "@entria/graphql-mongo-helpers";
+
+import { load } from "./EventLoader";
+
+import { nodeInterface, registerTypeLoader } from "../node/typeRegister";
+
+const EventType = new GraphQLObjectType({
+  name: "Event",
+  description: "event data",
+  fields: () => ({
+    id: globalIdField("Event"),
+    name: {
+      type: GraphQLString,
+      resolve: event => event.name,
+    },
+    start: {
+        type: GraphQLString,
+        resolve: event => event.start,
```

Comment:
if this is data try to return it already formated. It is better format on backend than on the frontend